### DISCUSSION TITLE
GATK-gCNV on Exome Data

### DISCUSSION THREAD

**Discussion by Faizal-Eeman** [uclahs-cds/pipeline-call-gSV#90] [Announcements]
@pboutros @tyamaguchi-ucla @yashpatel6 

https://www.nature.com/articles/s41588-023-01449-0

GATK has a gCNV algo to call common and rare germline variants. Also supports exome data!

We can consider applying this in the gSV pipeline which already uses DELLY CNV calling. Maybe not on the immediate plan but something to consider in the future, because this seems to require a set of control samples to build a model.

**Comment by pboutros** [comment]
Yup, worth creating an issue and we’ll get to this eventually!

________________________________

UCLA HEALTH SCIENCES IMPORTANT WARNING: This email (and any attachments) is only intended for the use of the person or entity to which it is addressed, and may contain information that is privileged and confidential. You, the recipient, are obligated to maintain it in a safe, secure and confidential manner. Unauthorized redisclosure or failure to maintain confidentiality may subject you to federal and state penalties. If you are not the intended recipient, please immediately notify us by return email, and delete this message from your computer.

**Comment by Faizal-Eeman** [comment]
Issue #91
